### PR TITLE
fix: change auto_logout value set logic

### PR DESCRIPTION
**Pull request by ggstone0523**
### Resolve: lablup/backend.ai-webui #1173 

**1. bugfix for auto logout feature**

- This bug is caused by changing the 'auto logout' setting on the 'UserSettings' page does not change value of 'this.auto_logout' accordingly.
- The reason why the value of 'this.auto_logout' does not change is that it continues to receive the value only from element 'autoLogout' of config.toml.

### CODE REVIEW

File: src/components/backend-ai-webui.ts [hunk 1]
```diff
@@ -306,7 +306,9 @@ export default class BackendAIWebUI extends connect(store)(LitElement) {
       this.connection_server = config.general.connectionServer;
       // console.log(this.connection_server);
     }
```
Comment (lizable) [inline src/components/backend-ai-webui.ts:308]: It works okay, but It seems too long to use ternary operation. How about using plain if-else statement for better readability?

**Review by lizable — CHANGES_REQUESTED** [review]
I'm not sure it's okay to use ternary operation in such a long statement. @ggstone0523
But All seems to work fine, so I want your opinion on this code.

**Comment by lizable** [comment]
LGTM!

**Comment by ggstone0523** [comment]
I also think so writing an if-else statement is a better way in terms of readability than using a ternary operation.

**Review by Sujin-Kim1 — APPROVED** [review]
LGTM

**Review by lizable — APPROVED** [review]

**Comment by inureyes** [comment]
LGTM

**Pull request merged**